api/tasks/runner.go

```diff
@@ -347,26 +348,31 @@ func (t *task) runGalaxy() error {
 	return cmd.Run()
 }
 
-func (t *task) listPlaybookHosts() error {
+func (t *task) listPlaybookHosts() (string, error) {
 	args, err := t.getPlaybookArgs()
 	if err != nil {
-		return err
+		return "", err
 	}
 	args = append(args, "--list-hosts")
 
 	cmd := exec.Command("ansible-playbook", args...)
 	cmd.Dir = util.Config.TmpPath + "/repository_" + strconv.Itoa(t.repository.ID)
 	cmd.Env = t.envVars(util.Config.TmpPath, cmd.Dir, nil)
 
+	var errb bytes.Buffer
+	cmd.Stderr = &errb
+
 	out, err := cmd.Output()
+	stderr := errb.String()
```

Comment:
Do we need this assignment if we don't use the variable in this function, can't we just return `errb.String()`?

Reply:
Sure we can.